omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -58,6 +58,12 @@
             # cleanup clutter
             delete "#{install_dir}/etc"
         elsif osx?
+            # Remove linux specific configs
+            conf_dir = "#{install_dir}/etc/conf.d"
```

Comment:
nit: I'd define `conf_dir` as `#{install_dir}/etc` and use it to delete `process-agent.conf.example`